### PR TITLE
Functions to retrieve gemmi atoms from gemmi model

### PR DESCRIPTION
**Pull request by fredericpoitevin**
Imported from cryonettorch with slight edits from Ariana's code.

### CODE REVIEW

**Review by ninamiolane — APPROVED** [review]
LGTM!

File: tests/test_atomic_models.py [hunk 1]
```diff
@@ -60,6 +62,48 @@ def test_read_atomic_model_from_cif(self):
         model = read_atomic_model(path)
         assert model.__class__ is gemmi.Model
 
+    def test_extract_gemmi_atoms(self):
+        """Check that extraction retrieves correct no. of chains."""
+        pdb_filename = "2dhb.pdb"
+        path = os.path.join(DATA, pdb_filename)
+        os.system(f"wget https://files.rcsb.org/download/{pdb_filename} -P {DATA}")
```
Comment (apeck12) [inline tests/test_atomic_models.py:69]: Would it make sense to `os.delete` the file at the end of the test?

**Review by apeck12 — APPROVED** [review]
Just the one (nonessential) question about deletion of the test file. Otherwise, no further questions -- it all looks good to me!

File: tests/test_atomic_models.py (hunk 1 above)
Comment (fredericpoitevin) [reply tests/test_atomic_models.py:71]: Good catch! Yes this is definitely a good idea

**Pull request merged**